### PR TITLE
gzip compression

### PR DESCRIPTION
**Pull request by ahsan13jan**

### CODE REVIEW

File: lib/fluent/plugin/out_lm.rb [hunk 1]
```diff
@@ -28,6 +29,8 @@ class LmOutput < BufferedOutput
 		
     config_param :force_encoding,  :string, :default => ""
 
+    config_param :compression,  :string, :default => ""
```
Comment (joch) [inline lib/fluent/plugin/out_lm.rb:32]: Any reason to not enable gzip by default?

Comment (ahsan13jan) [reply lib/fluent/plugin/out_lm.rb:32]: Not any, just to have an option to not gzip it, if the user wants to.

Comment (ahsan13jan) [reply lib/fluent/plugin/out_lm.rb:32]: And also, We have not tested yet! how much increase in cpu would it cost for k8s integration.

Comment (joch) [reply lib/fluent/plugin/out_lm.rb:32]: As we discussed during standup - I agree, let’s keep it off by default until we have tested the impact in k8s.

**Pull request merged**